WolvenKit.Modkit/Managers/ArchiveManager.cs

```diff
@@ -156,9 +158,14 @@ public override void LoadFromFolder(DirectoryInfo archiveDir)
                 return;
             }
 
+            var redModFolder = archiveDir.Name.Contains(Path.DirectorySeparatorChar + "mods" + Path.DirectorySeparatorChar);
+
             IsManagerLoading = true;
 
-            var archiveFiles = Directory.GetFiles(archiveDir.FullName, "*.archive").ToList();
+            // TODO: this will still load things in sub-directories beyond "archive" for REDMod mods
```

Comment:
Does it matter if we grab non-enabled archives? It'd save users the trouble of having to enable things just to mod them unless it throws something else off.

Reply:
For me, I would say yes, it matters. I'm not the only one who sometimes throws old archives for some appearance mods into subfolders to quickly disable them for determining which I like more. Disabling the subfolders prevents random ones the game isn't even loading from even being shown in the AB.

For example, imagine the mess if someone runs the export script and it starts grabbing files that the game isn't even using and shoving it in the material repo. Or setting up an NPV and the same happens. And so on